client/my-sites/plans/current-plan/style.scss

```diff
@@ -32,13 +19,19 @@
 }
 
 .current-plan__header-icon {
-	margin: 0 auto;
 	width: 80px;
 	height: 80px;
+	float: left;
+	margin-left: 20px;
+	margin-right: 40px;
+	display: flex;
+	flex-direction: row;
 }
 
 .current-plan__header-copy {
 	width: 100%;
+	text-align: left;
+	margin-top: -20px;
```

Comment:
Having the necessity of using negative margins is usually not a good practice and indicates we've done something wrong with our styles. Can we do anything to avoid it?

Reply:
good point ...again! :)